shared-lib/spiutils/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+
+name = "spiutils"
+version = "0.1.0"
+authors = [ "Oskar Senft <osk@google.com>" ]
+license = "MIT"
+keywords = ["spi-utils"]
+description = """
+Utilities for SPI
```

Comment:
I'm not sure this description is very good -- this crate is probably going to be somewhat Titan-specific, correct?

I think we can remove it, unless you expect this to be uploaded to `crates.io`.

Reply:
I was expecting the crate to be independent from Titan. Two things I know I want in there:
- SPI flash commands (done)
- SPI flash ID generation like JEDEC ID and SFDP.

Reply:
Perhaps the crate name should be something like `spi_flash_decls`? `spiutils` isn't descriptive enough for `crates.io`.

Reply:
Do you want to give this a name more suitable for `crates.io` or submit as-is?